### PR TITLE
security quiz questions on the CIA Triad

### PR DESCRIPTION
**Pull request by maame-deveer**
## Summary of changes

Added three quiz questions to the security quiz. they were on the CIA triad, I looked through the available questions and saw there was none on the CIA triad.

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/security-quiz.ts [hunk 1]
```diff
@@ -564,14 +564,66 @@ const securityQuiz = [
...
     Distractor1: "Ubuntu",
     Distractor2: "Windows",
     Distractor3: "MacOS",
     Explanation:
       "Kali Linux is a Debian-based Linux distribution aimed at advanced penetration testing and security auditing.",
     Link: "https://www.freecodecamp.org/news/how-to-install-kali-linux/"
+  },
+  {
+    Question: "What is the CIA Triad?",
+    Answer: "They are the guiding principles of information security.",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:579]:
```suggestion
    Answer: "A set of guiding principles on information security.",
```

File: src/data/security-quiz.ts [hunk 2]
```diff
@@ -564,14 +564,66 @@ const securityQuiz = [
...
     Answer: "Kali Linux",
     Distractor1: "Ubuntu",
     Distractor2: "Windows",
     Distractor3: "MacOS",
     Explanation:
       "Kali Linux is a Debian-based Linux distribution aimed at advanced penetration testing and security auditing.",
     Link: "https://www.freecodecamp.org/news/how-to-install-kali-linux/"
+  },
+  {
+    Question: "What is the CIA Triad?",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:578]:
```suggestion
    Question: "In CyberSecurity, what is the CIA Triad?",
```

File: src/data/security-quiz.ts [hunk 3]
```diff
@@ -564,14 +564,66 @@ const securityQuiz = [
...
+    Answer: "They are the guiding principles of information security.",
+    Distractor1: "The name of a black hat hacker group.",
+    Distractor2: "Created by the CIA to regulate security issues.",
+    Distractor3: "A standard policy for information security.",
+    Explanation:
+      "The CIA Triad serves as guiding principles or goals for information security for organizations and individuals to keep information safe from prying eyes.",
+    Link: "https://www.freecodecamp.org/news/the-cia-triad-confidentiality-integrity-and-availability-explained/"
+  },
+  {
+    Question: "What does CIA Triad stand for?",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:588]:
```suggestion
    Question: "In CyberSecurity, what does the CIA Triad stand for?",
```

File: src/data/security-quiz.ts [hunk 4]
```diff
@@ -564,14 +564,66 @@ const securityQuiz = [
...
+    Answer: "Confidentiality, Integrity and Availability",
+    Distractor1: "Central Intelligence Agency",
+    Distractor2: "Compromise, Install and Avoid",
+    Distractor3: "Certified Internal Auditor",
+    Explanation:
+      "CIA is the abbreviated form of Confidentillity, Integrity and Availability which makes up the CIA triad.",
+    Link: "https://www.freecodecamp.org/news/the-cia-triad-confidentiality-integrity-and-availability-explained/"
+  },
+  {
+    Question: "Confidentiality in the CIA Triad is about",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:598]:
```suggestion
    Question: "In CyberSecurity, how does the Confidentiality principle work in the CIA Triad?",
```

File: src/data/security-quiz.ts [hunk 5]
```diff
@@ -564,14 +564,66 @@ const securityQuiz = [
...
+      "ensuring access to data is restricted to only the intended audience and not others.",
+    Distractor1: "maintaining the accuracy, and completeness of data.",
+    Distractor2: "being discreet as a cybersecurity proffessional.",
+    Distractor3: "information being available when it is needed.",
+    Explanation:
+      "Confdentiality is a key principle in information seurity and ensuring only authorized people have access to sensitive information.",
+    Link: "https://www.freecodecamp.org/news/the-cia-triad-confidentiality-integrity-and-availability-explained/"
+  },
+  {
+    Question: "Integrity in the CIA Triad refers to",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:609]:
```suggestion
    Question: "In CyberSecurity, how does the Integrity principle work in the CIA Triad?",
```

File: src/data/security-quiz.ts [hunk 6]
```diff
@@ -564,14 +564,66 @@ const securityQuiz = [
...
+    Answer: "maintaining the accuracy, and completeness of data.",
+    Distractor1: "being truthful about security processes.",
+    Distractor2: "having integrity as a cybersecurity proffessional.",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:612]:
```suggestion
    Distractor2: "having integrity as a cybersecurity professional.",
```

File: src/data/security-quiz.ts [hunk 7]
```diff
@@ -564,14 +564,66 @@ const securityQuiz = [
...
+    Distractor3: "ensuring users have access to the right information.",
+    Explanation:
+      "Ensuring Integrity of information means to prevent hackers and other unauthorized entities from modifying information.",
+    Link: "https://www.freecodecamp.org/news/the-cia-triad-confidentiality-integrity-and-availability-explained/"
+  },
+  {
+    Question: "Availability in the CIA Triad is about",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:619]:
```suggestion
    Question: "In CyberSecurity, how does the Availability principle work in the CIA Triad?",
```

File: src/data/security-quiz.ts [hunk 8]
```diff
@@ -564,14 +564,66 @@ const securityQuiz = [
...
+    Answer: "information being available when it is needed.",
+    Distractor1: "being truthful about security processes.",
+    Distractor2: "having integrity as a cybersecurity proffessional.",
+    Distractor3: "ensuring users have access to the right information.",
+    Explanation:
+      "The availability of information when needed is crucial and ensuring that is an important part of cybersecurity.",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:625]:
```suggestion
      "To ensure high data availability, you must maintain a correctly functioning hardware and software and provide adequate bandwidth. ",
```

File: src/data/security-quiz.ts [hunk 9]
```diff
@@ -564,14 +564,66 @@ const securityQuiz = [
...
+    Distractor3: "ensuring users have access to the right information.",
+    Explanation:
+      "Ensuring Integrity of information means to prevent hackers and other unauthorized entities from modifying information.",
+    Link: "https://www.freecodecamp.org/news/the-cia-triad-confidentiality-integrity-and-availability-explained/"
+  },
+  {
+    Question: "Availability in the CIA Triad is about",
+    Answer: "information being available when it is needed.",
+    Distractor1: "being truthful about security processes.",
+    Distractor2: "having integrity as a cybersecurity proffessional.",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:622]:
```suggestion
    Distractor2: "having integrity as a cybersecurity professional.",
```

File: src/data/security-quiz.ts [hunk 10]
```diff
@@ -564,14 +564,66 @@ const securityQuiz = [
...
+    Distractor3: "ensuring users have access to the right information.",
+    Explanation:
+      "Ensuring Integrity of information means to prevent hackers and other unauthorized entities from modifying information.",
+    Link: "https://www.freecodecamp.org/news/the-cia-triad-confidentiality-integrity-and-availability-explained/"
+  },
+  {
+    Question: "Availability in the CIA Triad is about",
+    Answer: "information being available when it is needed.",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:620]:
```suggestion
    Answer: "ensuring that information is available when it is needed.",
```

File: src/data/security-quiz.ts [hunk 11]
```diff
@@ -564,14 +564,66 @@ const securityQuiz = [
...
+  },
+  {
+    Question: "Confidentiality in the CIA Triad is about",
+    Answer:
+      "ensuring access to data is restricted to only the intended audience and not others.",
+    Distractor1: "maintaining the accuracy, and completeness of data.",
+    Distractor2: "being discreet as a cybersecurity proffessional.",
+    Distractor3: "information being available when it is needed.",
+    Explanation:
+      "Confdentiality is a key principle in information seurity and ensuring only authorized people have access to sensitive information.",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:605]:
```suggestion
      "Confidentiality is a key principle in information security and ensuring only authorized people have access to sensitive information.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by maame-deveer** [comment]
I have applied all the suggested changes. thank you

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**